### ISSUE TITLE
Updated analysis: Update PFA EPN subtyping for EZHIP over-expression

### ISSUE THREAD

**Issue by jharenza** [AlexsLemonade/OpenPBTA-analysis#1049]
#### What analysis module should be updated and why?
molecular-subtyping-EPN

Quick update to include over-expression of _EZHIP_ (_CXorf67_) alone, as a biomarker of PFA EPN, according to the following publication:
[EZHIP is a specific diagnostic biomarker for posterior fossa ependymomas, group PFA and diffuse midline gliomas H3-WT with EZHIP overexpression
](https://actaneurocomms.biomedcentral.com/articles/10.1186/s40478-020-01056-8) since 1q gain is not present in _all_ PFA EPNs ([ref](https://actaneurocomms.biomedcentral.com/articles/10.1186/s40478-019-0820-5) and [ref](https://ascopubs.org/doi/10.1200/JCO.18.01765)). 

A quick look at RNA-Seq collapsed data and pedcbio tells me we will have more samples which can fit into this subtype.
<img width="1127" alt="Screen Shot 2021-04-30 at 7 55 35 PM" src="https://user-images.githubusercontent.com/207034/116764151-12580100-a9ee-11eb-9b65-f568a0b47cfb.png">

#### What changes need to be made? Please provide enough detail for another participant to make the update.
Add `row["CXorf67_expr_zscore"]>3) or` to logic for classifying PFA EPN within the below code:

```
def prioritizing_PT_EPN(row, sample_list):
    if((row["CXorf67_expr_zscore"]>3 and row["1q_gain"]>0) or
        (row["TKTL1_expr_zscore"]>3  and row["1q_gain"]>0)):
        sample_list.append(row["sample_id"])
        return("EPN, PF A")
```

#### What input data should be used? Which data were used in the version being updated?
RNA-Seq collapsed matrices

#### When do you expect the revised analysis will be completed?
1-2 weeks

#### Who will complete the updated analysis?
@kgaonkar6 

Also wanted to ask @jashapiro and @jaclyn-taroni why we had settled on a z-score >3 for identifying over-expression? Pedcbio uses >2, but I have a feeling we did this to have higher confidence. Curious.

**Comment by jaclyn-taroni** [comment]
> Also wanted to ask @jashapiro and @jaclyn-taroni why we had settled on a z-score >3 for identifying over-expression? Pedcbio uses >2, but I have a feeling we did this to have higher confidence. Curious.

I believe picked 3 across the board (i.e., for all subtyping modules where overexpression is included, not just EPN) to be very stringent.

**Comment by kgaonkar6** [comment]
Closed with #1068
